src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -845,7 +845,6 @@ public function providePathsForMakePathRelative()
             array('/var/lib/symfony/src/Symfony/', '/var/lib/symfony/src/Symfony/Component/', '../'),
```

Comment:
Can we have simple case `array('aa', 'bb', '../aa/'),` as this is example of what `installRequirementsFile` from `Sensio\Bundle\DistributionBundle\Composer\ScriptHandler` is breaking in composer script in latest Symfony?